<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/backgr.jpg);</string>
   </property>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#Title{
background-color:transparent; 
background:none; 
border:none ; 
color:white;

}</string>
    </property>
    <property name="text">
     <string>Create an account</string>
    </property>
   </widget>
   <widget class="QLabel" name="Email">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#Email{ 
background-color:transparent; 
background:none; 
border:none ; 
color:white;
}</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="Username">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#Username{ 
background-color:transparent; 
background:none; 
border:none ; 
color:white;
}</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="Password">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#Password{ 
background-color:transparent; 
background:none; 
border:none ; 
color:white;
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="Confirm">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#Confirm{ 
background-color:transparent; 
background:none; 
border:none ; 
color:white;
}</string>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailLine">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#EmailLine{ 
background-color:transparent; 
color:white; 
border:none; 
border-radius:12px; 

}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UsenameLine">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#UsenameLine{ 
background-color:transparent; 
color:white; 
border:none; 
border-radius:12px; 

}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordLine">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#PasswordLine{ 
background-color:transparent; 
color:white; 
border:none; 
border-radius:12px; 

}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfirmLine">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#ConfirmLine{ 
background-color:transparent; 
color:white; 
border:none; 
border-radius:12px; 

}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="SignUpButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#SignUpButton{
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(23, 37, 104, 255), stop:1 rgba(255, 255, 255, 255));
border:none ; 
color:white;
font: 10pt &quot;Segoe UI&quot;;
border-radius: 8px;
} 
#SignUpButton:hover{ 
	font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px;
}</string>
    </property>
    <property name="text">
     <string>SIGN UP</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>23</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/User (2).png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>23</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/PadLock (2).png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>23</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/PadLock (2).png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Email (2).png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>250</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/LogoImage.png); 
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>650</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#BackButton{
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(23, 37, 104, 255), stop:1 rgba(255, 255, 255, 255));
border:none ; 
color:white;
font: 10pt &quot;Segoe UI&quot;;
border-radius: 8px;
} 
#BackButton:hover{ 
	font: 700 9pt &quot;Segoe UI&quot;;
border-radius: 12px;
}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
